lib/minitest_to_rspec/subprocessors/defn.rb

```diff
@@ -0,0 +1,44 @@
+require_relative "../model/defn"
+require_relative "base"
+
+module MinitestToRspec
+  module Subprocessors
+    # Minitest tests can be defined as methods using names beginning with 'test_'.
+    # Process those tests into RSpec `it` example blocks.
+    class Defn < Base
+      def initialize(sexp, rails, mocha)
+        super(rails, mocha)
+        @rails = rails
+        @mocha = mocha
+        @exp = Model::Defn.new(sexp)
+        sexp.clear
+      end
+
+      # Given a `Model::Defn`, returns a `Sexp`
```

Comment:
s/Given/Using

"Given" implies a method a method argument. This is probably my fault, copied from elsewhere.